testdata/project-v4-with-grafana/.devcontainer/post-install.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright 2024 The Kubernetes Authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.24.0/kind-linux-amd64
```

Comment:
If we use a tag then we need keep it updated
So, since it is only for local dev env
We should use latest

So that we do not have effort to keep those files updated
It will install always the latest version of kind and kubectl.

Reply:
Latest release may unexpectedly break the otherwise working devcontainer. Having specific version IMO would be a better approach.

Reply:
It is kind/kubectl it will not really break anything

Reply:
Done! Replaced custom version numbers with latest.